debian/rules

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/make -f
 
+CMAKE_FLAGS += -DDEB_BUILD_MULTIARCH=`dpkg-architecture -qDEB_BUILD_MULTIARCH` -DPORTABLE=yes
```

Comment:
what does the `dpkg-architecture -qDEB_BUILD_MULTIARCH` do?

Reply:
It's supposed to determine the architecture of the current host system where the debian package is created.

For instance, in my case, this results in:
```
$ dpkg-architecture --query DEB_HOST_MULTIARCH
x86_64-linux-gnu
```
So this should support the cross-compilation of this package.